src/site/content/en/fast/serve-images-with-correct-dimensions/index.md

```diff
@@ -121,6 +122,26 @@ If you'll be resizing many images, you may find it more convenient to use a
 script or service to automate the process. You can learn more about this in the
 Responsive Images guide.
 
+## Avoid layout shifts by specifying dimensions
+
+While this guide discusses image dimensions in the context of reducing the amount of unnecessary bytes downloaded, it's important to note that reserving the correct space for images in the layout is another crucial component of minimizing a page's [Cumulative Layout Shift](/cls/) metric. When serving images in HTML, be sure to use proper `width` and `height` attributes so that the browser knows how much space to allocate in the layout for the image:
+
+```html
+<img src="flower.jpg" width="640" height="480" alt="A picture of a siberian iris.">
+```
+
+Without these attributes, of the equivalent CSS sizing, the browser has no idea how much space the image will take up until it has loaded. This will cause layout shifts in the document, which can be frustrating to users when they "miss" their intended hit target and end up clicking on something else they didn't intend to during page load.re-run
```

Comment:
Typo of mine `of` - > `or`.

Also I think mis-clicks is only one impact of CLS. The "losing your place" is more frequent IMHO, more annoying, though at least is easier resolved. WDYT about this?:

```suggestion
Without these attributes, or the equivalent CSS sizing, the browser has no idea how much space the image will take up until it has loaded. This will cause layout shifts in the document, which can be frustrating to users when content moves after they have started consuming it. This can result in losing your place when reading, or to "miss" their intended hit target and end up clicking on something else they didn't intend to during page load.
```

Reply:
The subject shifts in the edited paragraph. I recommend:

"This can result in losing your place when reading, or to "miss" **your** intended hit target..."

Reply:
Good spot. Edited the suggestion the other way as more consistent to what you originally suggested.

Feel free to change to "your" if you prefer.